### PR TITLE
Improve contracts test cases

### PR DESCRIPTION
**Pull request by EricBlack**

### CODE REVIEW

File: AElf.Contracts.Consensus.DPoS.Tests/ContractTesterExtensions.cs [hunk 1]
```diff
@@ -671,5 +671,44 @@ public static async Task<long> GetLIBOffset(this ContractTester<DPoSContractTest
...
+            {
+                return user.Address;
+            }
+
+            public static implicit operator string(User user)
+            {
+                return user.PublicKey;
+            }
+        }
+        public static User GenerateNewUser(this ContractTester<DPoSContractTestAElfModule> starter)
```
Comment (loning) [inline AElf.Contracts.Consensus.DPoS.Tests/ContractTesterExtensions.cs]: starter is not used.

it is tester not starter

File: AElf.Types.CSharp.Tests/ConversionExtensionTests.cs [hunk 2]
```diff
@@ -10,20 +10,38 @@ namespace AElf.Types.CSharp.Tests
...
+            returnObj = (bool)bs.DeserializeToType(typeof(bool));
+            returnObj.ShouldBeFalse();
+        }
+
+        [Fact(Skip = "Deserialize data is not correct.")]
+        public void Deserialize_From_ByteString_To_Int()
+        {
+            var intValue = new IntValue() { Value = 36 };
+            var bs = intValue.ToByteString();
+            var returnObj = (int)bs.DeserializeToType(typeof(int));
```
Comment (loning) [inline AElf.Types.CSharp.Tests/ConversionExtensionTests.cs]: IntValue cannot deserialize to int，it should be IntValue

File: AElf.Types.CSharp.Tests/ConversionExtensionTests.cs [hunk 3]
```diff
@@ -10,20 +10,38 @@ namespace AElf.Types.CSharp.Tests
...
+            returnObj.ShouldBe(36);
         }
 
+        [Fact(Skip = "Deserialize data is not correct.")]
+        public void Deserialize_From_ByteString_To_String()
+        {
+            var stringValue = new StringValue() { Value = "test info" };
```
Comment (loning) [inline AElf.Types.CSharp.Tests/ConversionExtensionTests.cs]: deserialize to StringValue

File: AElf.Contracts.Consensus.DPoS.Tests/ContractTesterExtensions.cs (hunk 1 above)
Comment (EricBlack) [reply AElf.Contracts.Consensus.DPoS.Tests/ContractTesterExtensions.cs]: renamed

File: AElf.Types.CSharp.Tests/ConversionExtensionTests.cs (hunk 3 above)
Comment (EricBlack) [reply AElf.Types.CSharp.Tests/ConversionExtensionTests.cs]: updated.

File: AElf.Types.CSharp.Tests/ConversionExtensionTests.cs (hunk 2 above)
Comment (EricBlack) [reply AElf.Types.CSharp.Tests/ConversionExtensionTests.cs]: updated.

**Pull request merged**